src/kubectl.ts

```diff
@@ -126,11 +126,11 @@ async function invokeWithProgress(context: Context, command: string, progressMes
     });
 }
 
-async function invokeAsync(context: Context, command: string): Promise<ShellResult> {
+async function invokeAsync(context: Context, command: string, stdin: string): Promise<ShellResult> {
```

Comment:
`stdin` should be optional here I feel (since it is in the interface).

Reply:
removed.